### PR TITLE
CLI: Update --store help wording

### PR DESCRIPTION
**Pull request by BramGruneir**
commit 1: update the wording for `--store`
commit 2: put back in the default value for `--store`

Wording updates are from @jseldess

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4729)

### CODE REVIEW

File: cli/flags.go [hunk 1]
```diff
@@ -135,50 +135,58 @@ Adjusts the max idle time of the scanner. This speeds up the scanner on small
...
+` + wrapText(`
+In most cases, node-level attributes are preferable to store-level attributes.
+However, the "attr" field can be used to match capabilities for storage of
+individual databases or tables. For example, an OLTP database would probably
+want to allocate space for its tables only on solid state devices, whereas
 append-only time series might prefer cheaper spinning drives. Typical
 attributes include whether the store is flash (ssd), spinny disk (hdd), or
-in-memory (mem). Device attributes might also include speeds and other specs
-(7200rpm, 200kiops, etc.). For example:`) + `
+in-memory (men), as well as speeds and other specs. Attributes can be arbitrary
```
Comment (jseldess) [inline cli/flags.go]: Small typo: change "men" to "mem".

**Comment by jseldess** [comment]
LGTM

**Comment by petermattis** [comment]
LGTM

File: cli/flags.go (hunk 1 above)
Comment (BramGruneir) [inline cli/flags.go]: fixed

**Pull request merged**